organizing_hub/views/call_views.py

```diff
@@ -121,6 +129,46 @@ def can_make_call_for_campaign(user, call_campaign):
     return False
 
 
+def get_or_create_callers(caller_emails):
+    """
+    Get or create callers from a list of caller emails
+
+    Parameters
+    ----------
+    emails : list of str
+        List of email addresses
+
+    Returns
+        -------
+        Callers list
+            Returns list of ints corresponding to callprofile IDs
+    """
+
+    caller_ids = []
+
+    if caller_emails is not None and caller_emails != '':
+        caller_emails = [email.strip() for email in caller_emails.split(",")]
+
+        for email in caller_emails:
+            # TODO: support multiple accounts per email
```

Comment:
What does this mean? should there be a follow up ticket?

Reply:
https://ourrevolution.atlassian.net/browse/TECH-1259

Will add in code